storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.12.rst

```diff
@@ -0,0 +1,33 @@
+===========================
+|Percona XtraBackup| 2.4.12
+===========================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.4.12 on
+June 21, 2018. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.12/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+All Percona software is open-source and free.
+
+Bugs fixed
+==========
+
+* ``xtrabackup --copy-back`` didn't read plugins folder location from  ``plugin-dir`` setting of the ``my.cnf`` configuration file. Bug fixed :jirabug:`PXB-1544`.
```

Comment:
This should be `plugin-load` instead of `plugin-dir`. Bug description is wrong as well.